nnpdf_data/nnpdf_data/commondata/CMS_Z0J_13TEV/metadata.yaml

```
@@ -0,0 +1,63 @@
+setname: "CMS_Z0J_13TEV"
+
+version: 1
+version_comment: "Initial implementation"
+
+# References
+arXiv:
+  url: "https://arxiv.org/pdf/2205.04897.pdf"
+  journal: Eur.Phys.J.C 83 (2023) 628
+iNSPIRE:
+  url: "https://inspirehep.net/literature/2079374"
+hepdata:
+  url: "https://www.hepdata.net/record/ins2079374"
+  version: 1
+
+nnpdf_metadata:
+  nnpdf31_process: "DY NC"
+  experiment: "CMS"
+
+implemented_observables:
+- observable_name: "PT-M"
+  observable:
+    description: "CMS 13 TeV, Drell-Yan Transverse Momentum Distribution"
+    label: CMS $Z$ $p_T$ 13 TeV $(p_T^{ll},m_{ll})$
+    units: "[pb/GeV]"
+
+  ndata: 95
+  tables: ["50-76","76-106","106-170","170-350","350-1000"]
+  process_type: DY_NC_PT
+
+  plotting:
+    dataset_label: CMS $Z$ $p_T$ 13 TeV $(p_T^{ll},M_{ll})$
+    y_label: $d\sigma/dp_T(\\ell\\ell)$ (pb/GeV)
+    figure_by:
+    - m_ll
+    x_scale: log
+    plot_x: pT
+  kinematic_coverage: [pT, m_ll]
+  kinematics:
+    variables:
+      pT:
+         description: "lepton pair transverse momentum"
+         label: '$p_T$'
+         units: "GeV"
+      m_ll:
+         description: "Z boson mass squared"
+         label: '$m^2_{\ell \ell}$'
+         units: "GeV^2"
```

```suggestion
         label: '$m_{\ell \ell}$'
         units: "GeV"
```

Not sure whether I broke this during the rebase.